app/migrations/Version20210330075323.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\PreUpAssertionMigration;
+
+final class Version20210330075323 extends PreUpAssertionMigration
+{
+    protected function preUpAssertions(): void
+    {
+        $tableName = $this->getPrefixedTableName('pages');
```

Comment:
It's funny that the Page::TABLE_NAME was added in this PR but is not being used here.
```suggestion
        $tableName = $this->getPrefixedTableName(Page::TABLE_NAME);
```
Plus add the use statement.